### ISSUE TITLE
feature: Disable grafana dashboards explicitly in values.yaml

### ISSUE THREAD

**Issue by marcelpuchol-st** [VictoriaMetrics/helm-charts#1851]
**Chart name and version, where you feel a lack of requested feature**
chart: victoria-metrics-k8s-stack
version: v0.29.1

**Is your feature request related to a problem? Please describe.**
Right now we can not disable some default dashboards if the scrapping is enabled. For instance, if we want to scrape `kubelet` but we don't want the `kubelet` grafana dashboard, we can not disable it because:
1. Condition value is set to `true` in https://github.com/VictoriaMetrics/helm-charts/blob/f502b626a986c22ab6bf68f1df4b39a67dfcdaf4/charts/victoria-metrics-k8s-stack/files/dashboards/generated/kubelet.yaml#L8
1. The condition present in https://github.com/VictoriaMetrics/helm-charts/blob/f502b626a986c22ab6bf68f1df4b39a67dfcdaf4/charts/victoria-metrics-k8s-stack/templates/grafana/dashboard.yaml#L27 is always `true` because the condition set in previous point is `true`

**Describe the solution you'd like**
If we set:
```yaml
defaultDashboards:
  enabled: true
  dashboards:
    kubelet:
      enabled: false
```
The `kubelet` dashboard shouldn't be created.

**Describe alternatives you've considered**
Couldn't find any alternative to disable some default dashboards.

**Proposal**
This could be fixed by changing the condition present in https://github.com/VictoriaMetrics/helm-charts/blob/f502b626a986c22ab6bf68f1df4b39a67dfcdaf4/charts/victoria-metrics-k8s-stack/templates/grafana/dashboard.yaml#L27 by:
```
{{- $dashboardCondition := and $dashboardData.condition (dig $dashboardName "enabled" true $.Values.defaultDashboards.dashboards) -}}
```

**Comment by AndrewChubatiuk** [comment]
hey @marcelpuchol-st 
there were requests to enable dashboards even if monitoring for component is disabled, where your suggestion will not work
will think about a better solution for this

**Comment by AndrewChubatiuk** [comment]
merged a change, starting next release dashboard condition, like `($Values.kubelet).enabled` for kubelet dashboard will be ignored if `.Values.defaultDashboards.dashboards.kubelet.enabled` is defined

**Comment by AndrewChubatiuk** [comment]
feature is available in 0.30.3

**Comment by marcelpuchol-st** [comment]
Great! Thanks!
